### PR TITLE
Show user detail template

### PR DESCRIPTION
**Pull request by milaendo**
added show page for userDetail and test for route and content on the page #657

### CODE REVIEW

**Comment by velkymx** [comment]
ErrorException (E_ERROR)
Call to a member function format() on null (View: /opt/processmaker/resources/views/admin/users/show.blade.php)
loggedin_at

File: resources/views/admin/users/show.blade.php [hunk 1]
```diff
@@ -9,7 +9,103 @@
...
+        </div>
+      </div>
+    </div>
 @endsection
 
 @Section('js')
+@endsection
+
+@Section('css')
+<link href="https://fonts.googleapis.com/css?family=Open+Sans:400,600,700,800" rel="stylesheet">
```
Comment (velkymx) [inline resources/views/admin/users/show.blade.php]: Remove this

File: tests/Feature/Admin/UserTest.php [hunk 2]
```diff
@@ -83,10 +83,10 @@ public function testShowRoute()
       $user_uuid = factory(User::class)->create()->uuid_text;
       // get the URL
       $response = $this->webCall('GET', '/admin/users/'. $user_uuid);
-
       $response->assertStatus(200);
       // check the correct view is called
       $response->assertViewIs('admin.users.show');
+      $response->assertSee('Groups');
```
Comment (velkymx) [inline tests/Feature/Admin/UserTest.php]: Make this test generate a user and then assertSee the user's name on the page.

File: resources/views/admin/users/show.blade.php [hunk 3]
```diff
@@ -9,7 +9,103 @@
...
+            </tr>
+          </thead>
+          <tbody>
+            @forelse ($user->members() as $group_member)
+            <tr>
+              <td>{{$group_member->group->name}}</td>
+              <td>{{$group_member->group->status}}</td>
+              <td class="text-right"><i class="fas fa-trash-alt text-secondary"></i></td>
+            </tr>
+            @empty
```
Comment (velkymx) [inline resources/views/admin/users/show.blade.php]: Remove the groups table entirely if there are not groups found. Put it inside of a card with the text "User is not a member of any groups" centered with padding (See Zeplin).

File: resources/sass/app.scss [hunk 4]
```diff
@@ -68,6 +68,57 @@ html {
   margin: 15px 5px
 }
 
+.avatar-small {
```
Comment (velkymx) [inline resources/sass/app.scss]: This is duplicated from the nav bar scope. Can we combine into one?

File: database/factories/UserFactory.php [hunk 5]
```diff
@@ -30,6 +30,7 @@
         'birthdate' => $faker->dateTimeThisCentury->format('Y-m-d'),
         'timezone' => $faker->timezone,
         'language' => 'us_en',
+        'loggedin_at' => $faker->dateTimeThisCentury->format('Y-m-d'),
```
Comment (velkymx) [inline database/factories/UserFactory.php]: Add time to this

File: resources/views/admin/users/show.blade.php (hunk 1 above)
Comment (KillahDillah) [reply resources/views/admin/users/show.blade.php]: removed

**Pull request merged**